Expensify/App
# New line indicator working fine

### Details

Now new line indicator shows up when opening report by URL.
### Fixed Issues

$ https://github.com/Expensify/App/issues/6471

### Tests


1. Copy the report url.
2. Close the app.
3. Go to device 2 and send few messages to that report.
4. Go back to device 1 and open that report via url.
5. Note that now New line indicator shows up.

### QA Steps

**Web, Mobile Web**:
1. Copy the report url.
2. Close the app.
3. Go to device 2 and send few messages to that report.
4. Go back to device 1 and open that report via url.
5. Note that now New line indicator shows up.

**Desktop**:
1. Go to device 1 (Desktop app) and open a report.
2. Run command+R to reload the Desktop app.
3. While it is reloading, quickly go to device 2 and send few messages to that report. (We are simulating the report opened directly by URL within desktop app)
4. At this point we should have new pending messages when desktop app is about to load.
5. The reload will show up the opened report with the pending messages.
6. Note that now New line indicator shows up.

**Android,iOS**:
1. Go to device 2 and send few messages to a report.
2. Go to device 1 and open the app.
3. Within device 1 open that report.
4. Note that New line indicator shows up normally.

NOTE: We still have a delay on showing NEW marker, we will open a new issue separatelly for this.

### Tested On


### Videos

#### Web

https://user-images.githubusercontent.com/56851391/158258872-eea3894b-4d9b-4142-971a-3c758cf929a4.mp4

#### Mobile Web

https://user-images.githubusercontent.com/56851391/158266061-44d9f457-dad2-4d14-a52e-19e83c1073a9.mp4

#### Android

https://user-images.githubusercontent.com/56851391/158271126-6bea446f-3c99-4c17-8d57-bc3235814a08.mp4

#### Desktop

https://user-images.githubusercontent.com/56851391/158365442-b015b61c-e6f7-4adc-85db-538d55148390.mp4

#### iOS

https://user-images.githubusercontent.com/56851391/158302877-4751c8ff-1a39-4f92-8117-d210bb5d3016.mp4
